cdap-app-templates/cdap-etl/cdap-etl-proto/src/main/java/io/cdap/cdap/etl/proto/v2/ETLBatchConfig.java

```diff
@@ -121,6 +124,10 @@ public Integer getMaxConcurrentRuns() {
     return maxConcurrentRuns;
   }
 
+  public Boolean isPushdownEnabled() {
```

Comment:
We can't return just `boolean` here, can we? Also please double confirm `is` prefix works for non-primitive `Boolean`

Reply:
You're rignt, I don't need to return the Boolean object, the primitive type work just fine.